vendor/k8s.io/kubernetes/staging/src/k8s.io/client-go/util/certificate/certificate_manager.go

```diff
@@ -141,7 +141,6 @@ type manager struct {
 	certStore                Store
 	certAccessLock           sync.RWMutex
 	cert                     *tls.Certificate
-	rotationDeadline         time.Time
```

Comment:
@smarterclayton The changes in `certificate_manager.go`/`certificate_manager_test.go` look unrelated to introducing a new option. Are they really have to be a part of that PR or it was unintentionally?